apps/projects/app/components/App/App.js

```diff
@@ -489,7 +495,7 @@ class App extends React.PureComponent {
         <StyledAragonApp publicUrl={ASSETS_URL}>
           <BaseStyles />
           <ToastHub>
-            <Title text="Projects" />
+            <Title text="Projects 3" handleMenuPanelOpen={this.handleMenuPanelOpen} />
```

Comment:
Why Projects 3?

Reply:
typo??

Reply:
Leftover. Missed in pre-commit cleanup. Removed.